core/API/DataTableGenericFilter.php

```diff
@@ -120,6 +121,7 @@ private function getGenericFiltersHavingDefaultValues()
             foreach ($filters as $index => $filter) {
                 if ($filter[0] === 'Sort') {
                     $filters[$index][1]['filter_sort_column'] = array('string', $this->report->getDefaultSortColumn());
+                    $filters[$index][1]['filter_sort_order']  = array('string', $this->report->getDefaultSortOrder());
```

Comment:
Reports need to be able to specify a default sort order. Eg when defaultSortOrder should be `label, asc`